tests/testsuite/offline.rs

```diff
@@ -383,30 +383,39 @@ fn update_offline_not_cached() {
         )
         .file("src/main.rs", "fn main() {}")
         .build();
+
+    let location_searched_message;
+    if is_registry_set {
+        location_searched_message =
+            "`dummy-registry` index (which is replacing registry `crates-io`)";
+    } else {
+        location_searched_message = "crates.io index"
+    }
+
     p.cargo("update --offline")
         .with_status(101)
-        .with_stderr_data(
+        .with_stderr_data(&format!(
             "\
 [ERROR] no matching package named `bar` found
-location searched: registry `crates-io`
+location searched: {location_searched_message}
```

Comment:
We should try to keep snapshotting where possible.

Looking at this, I don't think the location is critical to the test and we could replace it with the `[..]`  wildcard

Reply:
Thanks for the feedback. I'm new to cargo test,  I want to make sure I fully understand your suggestion. Do I need to replace all instances of the hard-coded `location searched: XXX` with `[..]` in the `offline.rs` test cases, or only in this specific test case (`update_offline_not_cached`)?

Reply:
Only in this specific case to make it work in both situations